src/providers/trusted_service/context/mod.rs

```diff
@@ -10,7 +10,10 @@ use std::ptr::null_mut;
 use std::slice;
 use std::sync::Mutex;
 use ts_binding::*;
-use ts_protobuf::{CloseKeyIn, GetOpcode, OpenKeyIn, OpenKeyOut, SetHandle};
+use ts_protobuf::GetOpcode;
+
+// Bindgen fails to generate this constant.
+pub const PSA_KEY_ID_USER_MAX: u32 = 0x3fffffff;
```

Comment:
I don't know if you have access but it's also [here](https://docs.rs/psa-crypto/0.8.0/psa_crypto/types/key/constant.PSA_KEY_ID_USER_MAX.html)

Reply:
Ah, yes, nice!